### ISSUE TITLE
npm commands (ci vs install)

### ISSUE THREAD

**Issue by AndyWardUST** [jfrog/jfrog-cli#705]
**Is your feature request related to a problem? Please describe.**
When running npm ci through the jfrog-cli the log message reads Install not ci

**Describe the solution you'd like to see**
When running npm <x> through the jfrog-cli the log message reads <x> not Install

**Describe alternatives you've considered**
None, just cosmetic language in jfrog-cli

**Additional context**
Using Azure DevOps in my particular instance

**Comment by AndyWardUST** [comment]
For the solution, npm [x] should translate to log message npm [x]

**Comment by RobiNino** [comment]
Hi @AndyWardUST @s-geiger-si ,
Thanks for reporting these issues.

Each of these commands uses it's corresponding npm command, so these issues are purely cosmetic. 
This was caused by an issue in the logging mechanism of a common code base these commands share in the CLI. 

We are working on fixing this confusion.
Thanks

**Comment by RobiNino** [comment]
Hi @AndyWardUST @s-geiger-si ,
Thanks again for reporting this.
We recently released version 1.38.0 which includes a fix.
We'd appreciate your feedback for it.

**Comment by s-geiger-si** [comment]
I can confirm that the following is fixed in the new version:

* The help for `jfrog rt npm-ci` now shows correct output regarding `npm ci`:
    ```
    Name:
      jfrog rt npm-ci - Run npm ci.
    
    Usage:
      jfrog rt npmci [npm ci args] [command options]
    
    Arguments:
      npm ci args
       The npm ci args to run npm ci.
    ```

* Running `jfrog rt npm-ci` now shows:
    ```
    [Info] Running npm ci.
    ```

Thanks for fixing this so fast. This issue can be closed.
